docs/DeveloperGuide.md

```diff
@@ -210,30 +209,31 @@ Fig. 6 shows the correct project structure of the Core File Service used as exam
 
 A service can be only consumed in a plug-in project.
 
-1. Open the bundle manifest "**META-INF/MANIFEST.MF**" (relative to project root) of the plug-in
+1. Create new "Plug-in Project" called "**org.opentosca.core.file.service.consumer**"
+2. Open the bundle manifest "**META-INF/MANIFEST.MF**" (relative to project root) of the plug-in
    project that should consume a service.
-2. In the appearing Eclipse bundle manifest editor go to the tab "**Dependencies**"
-3. At "**Imported Packages**" click on "**Add...**", select the package that includes the interface of the needed service an click on "**OK**".
+3. In the appearing Eclipse bundle manifest editor go to the tab "**Dependencies**"
```

Comment:
No need to adapt all numbers. Markdown takes care at rendering. If you update, you better change all numbers to `1.`.

Reply:
Thanks for the hint. I assume, we can leave it as it is now, right?